meson_options.txt

```diff
@@ -1,3 +1,6 @@
+option('man', type : 'boolean',
+    value : 'false',
```

Comment:
In line with @eli-schwartz comment, I'd prefer this to be true.

Reply:
Yeah, I agree. I don't remember why I used 'false' in the first place.